### PR TITLE
Fix language server errors

### PR DESCRIPTION
**Pull request by matthdsm**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by sateeshperi — APPROVED** [review]

File: modules/nf-core/bwa/index/main.nf [hunk 1]
```diff
@@ -11,7 +11,7 @@ process BWA_INDEX {
     tuple val(meta), path(fasta)
 
     output:
-    tuple val(meta), path(bwa) , emit: index
+    tuple val(meta), path("./bwa") , emit: index
```
Comment (maxulysse) [inline modules/nf-core/bwa/index/main.nf:14]: `./bwa`?

Comment (matthdsm) [reply modules/nf-core/bwa/index/main.nf:14]: the language server thought the unquoted `bwa` was a variable

**Review by maxulysse — APPROVED** [review]

**Comment by matthdsm** [comment]
@adamrtalbot, added your fix here

**Comment by matthdsm** [comment]
Tests will never pass, because `bwa_index` is a dependency of @famosab 's `parabricks` test and can't run on CPU.

I guess this will be an issue for a great many of modules/subworkflows

**Comment by matthdsm** [comment]
Is somebody from @nf-core/core willing to take a chance on me and merge this without passing tests? I've already got 3 approvals, but the tests can never pass because of a GPU dependency.

**Comment by matthdsm** [comment]
Thanks @sateeshperi

**Pull request merged**